ci.js

```diff
@@ -0,0 +1,181 @@
+#! /usr/bin/env node
+
+const fs = require("fs").promises;
+const path = require("path");
+
+function printUsage() {
+    console.log(
+        `Usage: "ci check-consistency" or "ci bump-version %postfix%" or "ci get-version"`
+    );
+}
+
+let postfix;
+const mode = process.argv[2];
+
+switch (mode) {
+    case "get-version":
+        break;
+
+    case "bump-version":
+        postfix = process.argv[3];
+        if (!postfix) {
+            printUsage();
+            process.exit();
+        }
+        break;
+
+    case "":
+    case undefined:
+    case "check-consistency":
+        break;
+
+    default:
+        printUsage();
+        process.exit(0);
+}
+
+const pathToPackages = "./packages/";
+
+async function getFiles(currentPath, files) {
+    const entries = await fs.readdir(currentPath, { withFileTypes: true });
+
+    for (let file of entries) {
+        if (file.name === "node_modules") {
+            continue;
+        }
+
+        if (file.isDirectory()) {
+            await getFiles(`${currentPath}${file.name}/`, files);
+        } else if (file.name === "package.json") {
+            const packageJsonPath = path.join(
+                __dirname,
+                currentPath,
+                file.name
+            );
+            files.push(packageJsonPath);
+        }
+    }
+}
```

Comment:
With help of something like this we can avoid await inside the loop. Also I suggest to rename this function so it's more clear what it does, remove the need to mutate function argument and avoid using template-literal path construction (use join instead). I would also suggest to avoid using __dirname and use process.cwd() or something else because __dirname not supported by ECMAScript modules so it's not futureproof

const getPackageJsonsRecursive = async (currentPath) => (await Promise.all(
  (await fs.readdir(currentPath, { withFileTypes: true }))
    .filter(({name}) => name !== "node_modules" && (file.isDirectory() || file.name === "package.json"))
    .map(() => file.isDirectory() 
      ? getPackageJsonsRecursive(path.join(currentPath, file.name))
      : Promise.resolve([
          path.join(
            __dirname,
            currentPath,
            file.name
          )
        ])
    )
)).flat()